Neos.Flow/Tests/Unit/SignalSlot/DispatcherTest.php

```diff
@@ -144,7 +144,7 @@ public function dispatchPassesTheSignalArgumentsToTheSlotMethod(): void
         $dispatcher->connect('Foo', 'bar', $mockSlot, '', false);
         $dispatcher->injectObjectManager($mockObjectManager);
 
-        $dispatcher->dispatch('Foo', 'bar', ['foo' => 'bar', 'baz' => 'quux']);
+        $dispatcher->dispatch('Foo', 'bar', ['bar', 'quux']);
```

Comment:
PHP 8 named parameters...

Reply:
This reflects the added `array_values()` in `Dispatcher`? But why, that call would do exactly that. This test change shows a change in the API, which did not really happen…

Reply:
Hm, indeed - I guess that was a change from before the `array_values()` was added?
```suggestion
        $dispatcher->dispatch('Foo', 'bar', ['foo' => 'bar', 'baz' => 'quux']);
```

Reply:
I instead added a new test to cover that unnamed parameters also work.